### ISSUE TITLE
Switch program to flexible deployment of capital via world's best investors

### ISSUE THREAD

**Issue by pstan26** [blockstack/app-mining#222]
Below is an idea that _may_ do the following: (1) use simple subjective scoring (through documented experience and good judgment) as a replacement for objective scoring (tracking users, creating gameable tests), (2) removes onus of operations from PBC, (3) optimizes for quality over quantity of applications.

**What is the problem you are seeing? Please describe.**
The current state of App Mining is too mechanistic (doesn't capture the richness of human nature) and has many people playing to the test rather than setting out to found companies. Monthly payouts solve for sustaining operations but not with enough freedom to take bigger risks as a startup or with expert coaching of would-be founders.

**How is this problem misaligned with goals of app mining?**
We want a higher chance of blockstack apps getting built that people love, and presumably thats from aligned motivations & incentives leading to quality, rather than quantity of apps.

**What is the explicit recommendation you’re looking to propose?**
Hand over the App Mining treasury to a VC or Accelerator (USV, YC, a16z, YC China, Tech Stars, 500 Startups come to mind) and give them skin in the game (XX% of the treasury over a vested schedule with a practical investment requirement/schedule). Amount invested over time has reasonable flexibility, rather than a set amount per month.

**What is the dry run period (if any)**
Think you could test this out for a few months with a smaller amount of the App Mining Treasury and lengthen the cycle of funds distributed so proper due diligence can occur. 

**Describe your long term considerations in proposing this change. Please include the ways you can predict this recommendation could go wrong and possible ways mitigate.**
Biggest ways this goes wrong are: (1) firm misspends or doesn't invest funds aggressively/quickly enough...can be mitigated with proper vetting of firm with sophisticated investment thesis and a schedule for compulsory investing...ultimately the integrity of the fund and their skin in the game is important (2) funded startup pivots away from Stacks blockchain...clause to retract funding or maybe if the funding is just forwarding Stacking funds to the startup it can also be discontinued. (3) anything else?

I like this proposal because it doesn't overcomplicate things or try to delve too deeply into game theory. It _may_ satisfactorily answer the question "what would this look like if it were easy and effective?".  Let me know what you think :-)

**Comment by Walterion1** [comment]
That is a very good idea to put humans on the control for mining to decrease the gaming of the system. But the point that I'm sure you thought when you start the program is that humans works are not scalable (for thousands of apps) and open enough (to let work in a way that never happened before). I think we should not go with the path of VCs. Their method is old and too profit-based, and we are beginning a new era together that for being true, it needs making value more than profit. Putting people that their job is valuing "growth" is not the best fit for this program and Blockstack.
We are making new values that need time and a more importantly new mindset that we may not find easily out there. And even if it closes some doors on some gaming mechanisms, but it will open new doors on the issues that other older systems have.

I think separating Mining team form the core of Blockstack was a very nice move. I propose that instead of laying the program on a separate system, we are better to improve the team behind mining with some peoples as controllers and not judges. They may have a score or coefficient to control the output of the mechanic parts of the program until the day that it can be fully mechanical.
We have something similar for universities that you will score for exams (that people may cheat) and a human as a professor that chat with you and see your works, passion and may or may not accept you. I am sure academic people like @muneeb-ali will appreciate the idea.

This will ensure flexibility of enforcing Blockstack values on the program much faster than working with an outsider with different values. Also, it will have lesser costs and a much simpler workflow for PBC.

**Comment by pstan26** [comment]
> But the point that I'm sure you thought when you start the program is that humans works are not scalable (for thousands of apps) and open enough (to let work in a way that never happened before).
Yep, but maybe this is good for quality over quantity. Also for example YC reviews many thousands of startups every 6 months. If you have the right infrastructure this can scale.

> Their method is old and too profit-based, and we are beginning a new era together that for being true, it needs making value more than profit. 
Sure but I'd assume that would be what a foundation is for (grants for infrastructure etc). Also its not clear that "apps" can succeed in the long term if they are not profitable. Profit and value are not exclusive.
https://twitter.com/balajis/status/1219775777165791233?s=20 It is old system but incentives aligned fairly well all things considered.

>  I propose that instead of laying the program on a separate system, we are better to improve the team behind mining with some peoples as controllers and not judges. They may have a score or coefficient to control the output of the mechanic parts of the program until the day that it can be fully mechanical.
Some folks may disagree but it is not clear App Mining can ever be fully mechanical, so its tough to accept that premise as a starting point.

Again, I'm not fully sold on this proposal, but I think *could* solve a good chunk of issues.

**Comment by joberding** [comment]
I like this idea because it encourages the requirement for a business model or at least some semblance of a business model idea that you are testing with potential customers. YC doesn't expect founders to have completely figured out the business model but that they have thought about it and are testing something that can allow them to be profitable. 

There are issues, though. @Walterion1 has raised some. One problem is the requirement for scaling. Most VC's and accelerators are looking for a startup that can eventually scale unicorn style. I'm not sure this will be a fit for the App Mining program or the community ethos.

Perhaps a tier approach as posted by @sdeepak23 in #219 would work in combination with this idea.

There is much to think about and more work to do. I think it is important to know the needs of all stakeholders regarding the changes to App Mining. This will be the subject of my proposal.

**Comment by friedger** [comment]
Isn't Blockstack's `Signature Fund` for such kind of VC funding?
https://signaturefund.org/


JMOberding <notifications@github.com> schrieb am Mi., 22. Jan. 2020, 22:59:
